manheim_cloudmapper/port_check/pagerdutyv1.py

```diff
@@ -3,6 +3,7 @@
 import urllib3
 import json
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
logging.basicConfig should only be called once in a given python process, generally in the top-level script that's executed on the command line. So I'm pretty sure that it should only be in ``run_port_check.py`` and ``send_email.py``.